test/mocha/glob.js

```diff
@@ -21,4 +21,14 @@ describe("minify() with input file globs", function() {
         });
         assert.strictEqual(result.code, 'var print=console.log.bind(console);print("qux",function(n){return 3*n}(3),function(n){return n/2}(12)),function(n){print("Foo:",2*n)}(11);');
     });
+    it("should throw with non-matching glob string", function() {
+        assert.throws(function() {
+            Uglify.minify("test/input/issue-1242/blah.*");
+        }, "should throw file not found");
```

Comment:
I agree that `"?" should not match "/"` is a legitimate error.

> This test would have failed the same way before this PR. I added it here just as an extra safeguard.

By "safeguard" do you mean adding a test?